src/components/CytoscapeGraph/CytoscapeGraph.tsx

```diff
@@ -176,10 +179,43 @@ export class CytoscapeGraph extends React.Component<CytoscapeGraphProps, Cytosca
       }
     };
 
-    cy.on('tap', (evt: any) => {
+    const injectDoubleTap = fun => {
```

Comment:
`fun` is not the most intuitive variable. Is this 'fun'? no, it's not **fun**, but what does it represent?

Reply:
It is probably a function of some sort -- but what does it do?

Reply:
Something like `originalCallback`, `tapEventCallback` or renaming the function to `injectDoubleTapToClickEvent` would make it better?

Reply:
fun is done :(
`tapEventCallback` sounds good to me

Reply:
Changed.